CONTRIBUTING.md

```diff
@@ -4,7 +4,28 @@
 1. All python code must be python 2.6/2.7/3.6 compatible
 1. The code should follow linting from pylint
 
-## Unit tests
+## Developing locally
+
+Dependencies: 
+https://virtualenv.pypa.io/en/latest/installation.html
+
+```bash
+make install  # for centOS 8 dependencies
+CENTOS_VER=7 make install  # for centOS 7 dependencies
+CENTOS_VER=6 make install  # for centOS 6 dependencies
+```
+this command will create the virtual environment and installs needed dependencies.
+Now you have to activate the environment:
+```bash
+. .venv3/bin/activate
+```
+
+and you're ready to run tests with:
+```bash
+pytest
```

Comment:
When running pytest in the .venv2.7 and .venv3, I'm getting the following for many tests:
`E   ImportError: No module named pexpect`
and 
`E   ModuleNotFoundError: No module named 'pexpect'`

Shouldn't pexpect be added to requirements/centos{7,8}.requirements.txt? Then you could remove the pexpect installation trhough yum/dnf from the Dockerfiles.

Reply:
Need to think about how to make it better. From one side I would like to keep app and dev deps in a separate category. But I understand that it is going to fail for the local install case. I think I know how to address this:
extending the existing requirements (you could add there `-r requirementsfilename` and other deps)

Reply:
I'm getting `E   ImportError: No module named rpm` in `.venv2.7` now (not in `.venv3`).